test/util.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
```

Comment:
Heh, I wish I would have had this script a week or two ago! I actually wrote my own to correct all these references (see [2456984](https://github.com/rtomac/robotframework-selenium2library/commit/245698483ff52f6e9512a9314ff13c49c29f37a8)). I'm not opposed to adding this to correct any future mistakes, but I don't think it should be done in this PR. Mind moving this script out into its own PR?